packages/components/src/index.ts

```diff
@@ -25,7 +25,6 @@ export { SpinnerExample } from './spinner/example';
 export { GMClosureNotice } from './gm-closure-notice/gm-closure-notice';
 export { default as WordPressLogo } from './wordpress-logo';
```

Comment:
Should we move`WordPressLogo` along into the logos folder? also for `WooCommerceWooLogo` below